rust/src/cliwrap/dracut.rs

```diff
@@ -6,9 +6,11 @@ use crate::cliwrap::cliutil;
 
 /// Primary entrypoint to running our wrapped `dracut` handling.
 pub(crate) fn main(argv: &[&str]) -> Result<()> {
+    // Read environment variable to determine if we want to bypass the wrapped binary.
+    let use_wrap = std::env::var("DONT_USE_REAL_DRACUT").unwrap_or("none".to_string());
     // At least kdump.service runs dracut to generate a separate initramfs.
     // We need to continue supporting that.
-    if crate::utils::running_in_systemd() {
+    if crate::utils::running_in_systemd() || !use_wrap.to_lowercase().eq("true") {
```

Comment:
@cgwalters If I understand your comment correctly now. I had my logic wrong, I always want to go into here unless the env variable is set. Might need a better name for the ENV variable WDYT?